test/spec/jquery.spec.js

```diff
@@ -7,9 +7,10 @@ describe('backend + jQuery', function () {
   });
 
   it('should hit an end point when a stub does not exist', function (done) {
```

Comment:
I think this test needs to be renamed since we're testing for errors being thrown and not pass throughs.